static/locales/az.yaml

```diff
@@ -389,6 +389,8 @@ Profile:
   Profile Manager: ''
   Create New Profile: ''
   Edit Profile: ''
+  Upload Profile Picture: ''
```

Comment:
Please only change the language files that you actually want to change. That keeps the number of changes small and easier to review, also adding blank entries is not necessary, as vue-i18n falls back to en-US regardless of these blank entries or not.

Reply:
A quick clarifying question: should we just remove all changes to language files with blank entries, or remove all changes to any language file besides en-US?

Reply:
Remove blank entries
As without those the fallback still works
Less diff = easier review